template/registry_time.go

```diff
@@ -0,0 +1,98 @@
+package template
+
+import (
+	"fmt"
+	"time"
+
+	"golang.org/x/text/language"
+)
+
+// timeFunc is the implementation of the time function. Locale-sensitive time formatting.
+var timeRegistryFunc = RegistryFunc{
+	Format: timeFunc,
+}
+
+type timeOptions struct {
+	// (default is system default time zone or UTC)
```

Comment:
This comment does not align with the default value. Is the same applicable to `:date` comment?

```suggestion
	// (default is UTC)
```

Reply:
`timezone` option is not even defined for either `:date` or `:time`, so I was just assuming it would be appropriate to add it anyway.

Reply:
The comment does not reflect the implementation. :)

In our implementation, we never fall back to system time. It ALWAYS defaults to UTC.